### PR TITLE
feat: add details title configuration field

### LINKED ISSUE #565: Add an attribute to fieldsConfiguration that template the details title
- [ ] add `detailsTitleTemplate` under `fieldsConfiguration` field
- [ ] this attribute should handled the same way as urlThumbnailTemplate

### PR DESCRIPTION
**Pull request by sebbousquet**
- Fix #565

### CODE REVIEW

**Review by MAudelGisaia — APPROVED** [review]

File: src/models/models.ts [hunk 1]
```diff
@@ -122,6 +122,8 @@ export interface FieldsConfiguration {
     useHttpQuicklooks?: boolean;
     /** Whether the thumbnails are protected and the http call needs to be enriched with auth header */
     useHttpThumbnails?: boolean;
+    /** Template of detail title */
```
Comment (QuCMGisaia) [inline src/models/models.ts:125]: detail -> details ?

**Review by QuCMGisaia — APPROVED** [review]

File: src/models/models.ts (hunk 1 above)
Comment (sebbousquet) [reply src/models/models.ts]: Done

File: src/jsonSchemas/resultlistContributorConf.schema.json [hunk 2]
```diff
@@ -90,6 +90,10 @@
                 "iconColorFieldName": {
                     "description": "Field which value is transformed to a hex color (using an ArlasColorService) and associated to the icon color",
                     "type": "string"
+                },
+                "detailsTitleTemplate": {
+                    "description": "Template of detail title",
```
Comment (QuCMGisaia) [inline src/jsonSchemas/resultlistContributorConf.schema.json:95]: detail -> details

**Review by QuCMGisaia — APPROVED** [review]

File: src/jsonSchemas/resultlistContributorConf.schema.json (hunk 2 above)
Comment (sebbousquet) [reply src/jsonSchemas/resultlistContributorConf.schema.json:95]: 🙈

**Review by QuCMGisaia — APPROVED** [review]

**Pull request merged**